### PR TITLE
issue/1854: ie11 switched native [].find( to _.find([],

### PR DESCRIPTION
**Pull request by oliverfoster**
#1854

### CODE REVIEW

File: src/core/js/models/adaptModel.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ define([
                 }
 
                 // Check that property is trackable
-                var isTrackable = _.keys(model.changed).find(function(item, index) {
+                var isTrackable = _.find(_.keys(model.changed), function(item, index) {
                      return _.contains(_.result(this, 'trackable', []), item);
```
Comment (tomgreenfield) [inline src/core/js/models/adaptModel.js]: Could you cache `_.result(this, 'trackable', [])` outside of the loop please?

File: src/core/js/models/adaptModel.js [hunk 2]
```diff
@@ -96,8 +96,10 @@ define([
                 }
 
                 // Check that property is trackable
-                var isTrackable = _.find(_.keys(model.changed), function(item, index) {
-                     return _.contains(_.result(this, 'trackable', []), item);
+                var trackablePropertyeNames = _.result(this, 'trackable', []);
```
Comment (tomgreenfield) [inline src/core/js/models/adaptModel.js]: So much easier to read. Just a typo to fix in `trackablePropertyeNames`.

**Comment by oliverfoster** [comment]
@brian-learningpool could you give it another +1 so i can merge?

**Comment by oliverfoster** [comment]
ta dude

**Pull request merged**